### PR TITLE
book detail texts

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- book detail texts

## Trello Card
https://trello.com/c/2ktzeDg4/26-detalle-de-libro-i-textos

## Screenshot
Captura de pantalla 2019-12-04 a la(s) 5 26 39 p  m

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.book-container {
+  border: 1px solid #CDCDCD;
```
Comment (betoobernal) [inline scss/detail.scss]: use a variable for this color

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
-.title {
-  color: red;
+* {
+font-family: $font-family;
```
Comment (betoobernal) [inline scss/styles.scss]: missing indent space

**Review by betoobernal — APPROVED** [review]

File: scss/styles.scss (hunk 2 above)
Comment (jonnyalexbh) [reply scss/styles.scss]: ready

File: scss/detail.scss (hunk 1 above)
Comment (jonnyalexbh) [reply scss/detail.scss]: ready

**Pull request merged**